### ISSUE TITLE
Add Complete Intersection-over-union loss (CIOU Loss) implementation for fine-tuning MTCNN face detector.

### ISSUE THREAD

**Issue by LovePelmeni** [LovePelmeni/DeepfakeStream#25]
Create implementation of CIOU Loss (Complete Intersection-over-union Loss), considered to be fully enhanced version of Intersection Over Union Loss.  See paper: `https://arxiv.org/abs/1911.08287` and  torch documentation about CIOU Loss: `https://pytorch.org/vision/main/generated/torchvision.ops.complete_box_iou_loss.html` for more details
